src/encoded/static/components/biosample.js

```diff
@@ -34,9 +35,11 @@ class BiosampleComponent extends React.Component {
         // Build the text of the synchronization string
         let synchText;
         if (context.synchronization) {
+            const synchronizationTime = formatMeasurement(context.post_synchronization_time, context.post_synchronization_time_units);
+
             synchText = context.synchronization +
                 (context.post_synchronization_time ?
-                    ` + ${context.post_synchronization_time}${context.post_synchronization_time_units ? ` ${context.post_synchronization_time_units}` : ''}`
+                    synchronizationTime
```

Comment:
Doesn't this just print “synchronizationTime” or am I crazy? Doesn’t it have to be `{synchronizationTime}`? Or am I crazy?

Reply:
It is a string not in JSX html, so it is not needed. However, I update it because the '+' was not being printed as in prod.

Reply:
Oh, you’re right. I _am_ crazy.

Reply:
I updated the formatMeasurement structure. The Airbnb method in my experience works if you need "this" function context to be the parent function. Otherwise arrow function is better. I went with Redux. I did not go that route initially because other utils used "function" rather than arrow.